manifests/base/role.yaml

```diff
@@ -13,6 +13,7 @@ rules:
   - ''
   resources:
   - "secrets"
+  - "deployments"
```

Comment:
Why do we need this permission?

Reply:
I though it was a solution to an error it was getting. It turns out it was something else which was unrelated. Going to remote it.